tests/test_vscode_tasks.py

```
@@ -0,0 +1,93 @@
+    # Use a dummy path that does not exist
+    env = os.environ.copy()
+    env['BLENDER_PYTHON'] = 'Z:/not/a/real/path/python.exe'
+    # Run the script in PowerShell, capture output
+    if args[1].lower() == '-file' or script.lower().endswith('.ps1'):
+        # run the ps1 file via -File
+        ps_path = script.replace('${workspaceFolder}', os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+        ps_command = ["powershell", "-NoProfile", "-File", ps_path]
+    else:
+        ps_command = ["powershell", "-NoProfile", "-Command", script]
+
+    proc = subprocess.run(ps_command, env=env, capture_output=True, text=True, timeout=20)
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)